src/main/scala/edu/ie3/mobsim/MobilitySimulator.scala

```diff
@@ -119,15 +119,6 @@ final class MobilitySimulator(
       )
     }
 
-    /* Save occupancy of charging stations for csv output */
-    chargingStations.foreach(cs =>
-      ioUtils.writeEvcs(
-        cs,
-        chargingStationOccupancy,
```

Comment:
Value is not used anymore above :) 

Also in `exchangeEvMovementsWithSimona` we don't need to determine the return value anymore